modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Controller/PublisherMethodsController.cs

```diff
@@ -95,9 +96,9 @@ List<PublishNodesEndpointApiModel> request
         /// <summary>
         /// Handler for GetDiagnosticInfo DM
         /// </summary>
-        public async Task GetDiagnosticInfoAsync(PublishNodesEndpointApiModel request) {
-            await Task.Delay(0);
-            throw new MethodCallStatusException((int)HttpStatusCode.NotImplemented, "Not Implemented");
+        public async Task<List<JobDiagnosticInfoModel>> GetDiagnosticInfoAsync(PublishNodesEndpointApiModel request) {
```

Comment:
As defined in the spec the method does should not have any inputs. So we should remove `PublishNodesEndpointApiModel request` from it as it is also not used.

Reply:
removed. I kept it because it was present (but unused) in 2.5 dm code